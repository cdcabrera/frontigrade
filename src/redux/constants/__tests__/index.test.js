import {
  reduxTypes,
  aboutModalTypes,
  accountTypes,
  applicationStatusTypes,
  confirmationModalTypes,
  filterTypes,
  reportTypes,
  systemConfigTypes,
  toastNotificationTypes,
  userTypes
} from '..';

describe('ReduxTypes', () => {
  it('should export the same number of name-spaced types as imported', () => {
    expect(Object.keys(reduxTypes)).toHaveLength(9);
  });

  it('should return types that are defined', () => {
    Object.keys(reduxTypes).forEach(type => expect(reduxTypes[type]).toBeDefined());
  });

  it('should return types that match', () => {
    expect(reduxTypes.aboutModal).toEqual(aboutModalTypes);
    expect(reduxTypes.account).toEqual(accountTypes);
    expect(reduxTypes.applicationStatus).toEqual(applicationStatusTypes);
    expect(reduxTypes.confirmationModal).toEqual(confirmationModalTypes);
    expect(reduxTypes.filter).toEqual(filterTypes);
    expect(reduxTypes.report).toEqual(reportTypes);
    expect(reduxTypes.systemConfig).toEqual(systemConfigTypes);
    expect(reduxTypes.toastNotifications).toEqual(toastNotificationTypes);
    expect(reduxTypes.user).toEqual(userTypes);
  });
});
