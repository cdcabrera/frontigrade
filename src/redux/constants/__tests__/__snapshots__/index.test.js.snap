// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReduxTypes should have specific type properties: all redux types 1`] = `
Object {
  "aboutModalTypes": Object {
    "ABOUT_MODAL_HIDE": "ABOUT_MODAL_HIDE",
    "ABOUT_MODAL_SHOW": "ABOUT_MODAL_SHOW",
  },
  "accountTypes": Object {
    "ADD_ACCOUNT": "ADD_ACCOUNT",
    "ADD_ACCOUNT_SHOW": "ADD_ACCOUNT_SHOW",
    "ADD_ACCOUNT_WIZARD": "ADD_ACCOUNT_WIZARD",
    "ADD_ACCOUNT_WIZARD_STEP_ARN": "ADD_ACCOUNT_WIZARD_STEP_ARN",
    "ADD_ACCOUNT_WIZARD_STEP_POLICY": "ADD_ACCOUNT_WIZARD_STEP_POLICY",
    "ADD_ACCOUNT_WIZARD_STEP_ROLE": "ADD_ACCOUNT_WIZARD_STEP_ROLE",
    "DELETE_ACCOUNT": "DELETE_ACCOUNT",
    "DELETE_ACCOUNTS": "DELETES_ACCOUNTS",
    "DELETE_ACCOUNT_HIDE": "DELETE_ACCOUNT_HIDE",
    "DELETE_ACCOUNT_SHOW": "DELETE_ACCOUNT_SHOW",
    "EDIT_ACCOUNT_HIDE": "EDIT_ACCOUNT_HIDE",
    "EDIT_ACCOUNT_SHOW": "EDIT_ACCOUNT_SHOW",
    "GET_ACCOUNT": "GET_ACCOUNT",
    "GET_ACCOUNTS": "GET_ACCOUNTS",
    "GET_ACCOUNT_IMAGE": "GET_ACCOUNT_IMAGE",
    "GET_ACCOUNT_IMAGES": "GET_ACCOUNT_IMAGES",
    "GET_ACCOUNT_IMAGES_INSTANCES": "GET_ACCOUNT_IMAGES_INSTANCES",
    "GET_ACCOUNT_INSTANCES": "GET_ACCOUNT_INSTANCES",
    "INVALID_ACCOUNT_WIZARD_STEP_ARN": "INVALID_ACCOUNT_WIZARD_STEP_ARN",
    "INVALID_ACCOUNT_WIZARD_STEP_POLICY": "INVALID_ACCOUNT_WIZARD_STEP_POLICY",
    "INVALID_ACCOUNT_WIZARD_STEP_ROLE": "INVALID_ACCOUNT_WIZARD_STEP_ROLE",
    "UPDATE_ACCOUNT": "UPDATE_ACCOUNT",
    "UPDATE_ACCOUNTS": "UPDATE_ACCOUNTS",
    "UPDATE_ACCOUNT_FIELD": "UPDATE_ACCOUNT_FIELD",
    "UPDATE_ACCOUNT_HIDE": "UPDATE_ACCOUNT_HIDE",
    "UPDATE_ACCOUNT_IMAGE": "UPDATE_ACCOUNT_IMAGE",
    "UPDATE_ACCOUNT_IMAGES_INSTANCES": "UPDATE_ACCOUNT_IMAGES_INSTANCES",
    "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL": "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL",
    "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP": "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP",
  },
  "applicationStatusTypes": Object {
    "STATUS_5XX": "5XX",
    "default": "5XX",
  },
  "confirmationModalTypes": Object {
    "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
    "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
  },
  "default": Object {
    "aboutModal": Object {
      "ABOUT_MODAL_HIDE": "ABOUT_MODAL_HIDE",
      "ABOUT_MODAL_SHOW": "ABOUT_MODAL_SHOW",
    },
    "account": Object {
      "ADD_ACCOUNT": "ADD_ACCOUNT",
      "ADD_ACCOUNT_SHOW": "ADD_ACCOUNT_SHOW",
      "ADD_ACCOUNT_WIZARD": "ADD_ACCOUNT_WIZARD",
      "ADD_ACCOUNT_WIZARD_STEP_ARN": "ADD_ACCOUNT_WIZARD_STEP_ARN",
      "ADD_ACCOUNT_WIZARD_STEP_POLICY": "ADD_ACCOUNT_WIZARD_STEP_POLICY",
      "ADD_ACCOUNT_WIZARD_STEP_ROLE": "ADD_ACCOUNT_WIZARD_STEP_ROLE",
      "DELETE_ACCOUNT": "DELETE_ACCOUNT",
      "DELETE_ACCOUNTS": "DELETES_ACCOUNTS",
      "DELETE_ACCOUNT_HIDE": "DELETE_ACCOUNT_HIDE",
      "DELETE_ACCOUNT_SHOW": "DELETE_ACCOUNT_SHOW",
      "EDIT_ACCOUNT_HIDE": "EDIT_ACCOUNT_HIDE",
      "EDIT_ACCOUNT_SHOW": "EDIT_ACCOUNT_SHOW",
      "GET_ACCOUNT": "GET_ACCOUNT",
      "GET_ACCOUNTS": "GET_ACCOUNTS",
      "GET_ACCOUNT_IMAGE": "GET_ACCOUNT_IMAGE",
      "GET_ACCOUNT_IMAGES": "GET_ACCOUNT_IMAGES",
      "GET_ACCOUNT_IMAGES_INSTANCES": "GET_ACCOUNT_IMAGES_INSTANCES",
      "GET_ACCOUNT_INSTANCES": "GET_ACCOUNT_INSTANCES",
      "INVALID_ACCOUNT_WIZARD_STEP_ARN": "INVALID_ACCOUNT_WIZARD_STEP_ARN",
      "INVALID_ACCOUNT_WIZARD_STEP_POLICY": "INVALID_ACCOUNT_WIZARD_STEP_POLICY",
      "INVALID_ACCOUNT_WIZARD_STEP_ROLE": "INVALID_ACCOUNT_WIZARD_STEP_ROLE",
      "UPDATE_ACCOUNT": "UPDATE_ACCOUNT",
      "UPDATE_ACCOUNTS": "UPDATE_ACCOUNTS",
      "UPDATE_ACCOUNT_FIELD": "UPDATE_ACCOUNT_FIELD",
      "UPDATE_ACCOUNT_HIDE": "UPDATE_ACCOUNT_HIDE",
      "UPDATE_ACCOUNT_IMAGE": "UPDATE_ACCOUNT_IMAGE",
      "UPDATE_ACCOUNT_IMAGES_INSTANCES": "UPDATE_ACCOUNT_IMAGES_INSTANCES",
      "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL": "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL",
      "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP": "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP",
    },
    "applicationStatus": Object {
      "STATUS_5XX": "5XX",
      "default": "5XX",
    },
    "confirmationModal": Object {
      "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
      "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
    },
    "filter": Object {
      "GRAPH_OPENSHIFT_SET_FILTER_VALUE": "GRAPH_OPENSHIFT_SET_FILTER_VALUE",
      "GRAPH_RHEL_SET_FILTER_VALUE": "GRAPH_RHEL_SET_FILTER_VALUE",
      "TOOLBAR_ADD_FILTER": "TOOLBAR_ADD_FILTER",
      "TOOLBAR_CLEAR_DATE_TYPE": "TOOLBAR_CLEAR_DATE_TYPE",
      "TOOLBAR_CLEAR_FILTERS": "TOOLBAR_CLEAR_FILTERS",
      "TOOLBAR_REMOVE_FILTER": "TOOLBAR_REMOVE_FILTER",
      "TOOLBAR_SET_DATE_TYPE": "TOOLBAR_SET_DATE_TYPE",
      "TOOLBAR_SET_FILTER_TYPE": "TOOLBAR_SET_FILTER_TYPE",
      "TOOLBAR_SET_FILTER_VALUE": "TOOLBAR_SET_FILTER_VALUE",
      "TOOLBAR_SET_SORT_TYPE": "TOOLBAR_SET_SORT_TYPE",
      "TOOLBAR_TOGGLE_SORT_ASCENDING": "TOOLBAR_TOGGLE_SORT_ASCENDING",
    },
    "report": Object {
      "REPORT_EXPORT": "REPORT_EXPORT",
      "default": "REPORT_EXPORT",
    },
    "systemConfig": Object {
      "GET_SYSTEM_CONFIG": "GET_SYSTEM_CONFIG",
      "default": "GET_SYSTEM_CONFIG",
    },
    "toastNotifications": Object {
      "TOAST_ADD": "TOAST_ADD",
      "TOAST_PAUSE": "TOAST_PAUSE",
      "TOAST_REMOVE": "TOAST_REMOVE",
      "TOAST_RESUME": "TOAST_RESUME",
    },
    "user": Object {
      "USER_CREATE": "USER_CREATE",
      "USER_DELETE": "USER_DELETE",
      "USER_INFO": "USER_INFO",
      "USER_LOCALE": "USER_LOCALE",
      "USER_LOGIN": "USER_LOGIN",
      "USER_LOGOUT": "USER_LOGOUT",
      "USER_STORED_DATA": "USER_STORED_DATA",
      "USER_STORED_DATA_REMOVE": "USER_STORED_DATA_REMOVE",
    },
  },
  "filterTypes": Object {
    "GRAPH_OPENSHIFT_SET_FILTER_VALUE": "GRAPH_OPENSHIFT_SET_FILTER_VALUE",
    "GRAPH_RHEL_SET_FILTER_VALUE": "GRAPH_RHEL_SET_FILTER_VALUE",
    "TOOLBAR_ADD_FILTER": "TOOLBAR_ADD_FILTER",
    "TOOLBAR_CLEAR_DATE_TYPE": "TOOLBAR_CLEAR_DATE_TYPE",
    "TOOLBAR_CLEAR_FILTERS": "TOOLBAR_CLEAR_FILTERS",
    "TOOLBAR_REMOVE_FILTER": "TOOLBAR_REMOVE_FILTER",
    "TOOLBAR_SET_DATE_TYPE": "TOOLBAR_SET_DATE_TYPE",
    "TOOLBAR_SET_FILTER_TYPE": "TOOLBAR_SET_FILTER_TYPE",
    "TOOLBAR_SET_FILTER_VALUE": "TOOLBAR_SET_FILTER_VALUE",
    "TOOLBAR_SET_SORT_TYPE": "TOOLBAR_SET_SORT_TYPE",
    "TOOLBAR_TOGGLE_SORT_ASCENDING": "TOOLBAR_TOGGLE_SORT_ASCENDING",
  },
  "reduxTypes": Object {
    "aboutModal": Object {
      "ABOUT_MODAL_HIDE": "ABOUT_MODAL_HIDE",
      "ABOUT_MODAL_SHOW": "ABOUT_MODAL_SHOW",
    },
    "account": Object {
      "ADD_ACCOUNT": "ADD_ACCOUNT",
      "ADD_ACCOUNT_SHOW": "ADD_ACCOUNT_SHOW",
      "ADD_ACCOUNT_WIZARD": "ADD_ACCOUNT_WIZARD",
      "ADD_ACCOUNT_WIZARD_STEP_ARN": "ADD_ACCOUNT_WIZARD_STEP_ARN",
      "ADD_ACCOUNT_WIZARD_STEP_POLICY": "ADD_ACCOUNT_WIZARD_STEP_POLICY",
      "ADD_ACCOUNT_WIZARD_STEP_ROLE": "ADD_ACCOUNT_WIZARD_STEP_ROLE",
      "DELETE_ACCOUNT": "DELETE_ACCOUNT",
      "DELETE_ACCOUNTS": "DELETES_ACCOUNTS",
      "DELETE_ACCOUNT_HIDE": "DELETE_ACCOUNT_HIDE",
      "DELETE_ACCOUNT_SHOW": "DELETE_ACCOUNT_SHOW",
      "EDIT_ACCOUNT_HIDE": "EDIT_ACCOUNT_HIDE",
      "EDIT_ACCOUNT_SHOW": "EDIT_ACCOUNT_SHOW",
      "GET_ACCOUNT": "GET_ACCOUNT",
      "GET_ACCOUNTS": "GET_ACCOUNTS",
      "GET_ACCOUNT_IMAGE": "GET_ACCOUNT_IMAGE",
      "GET_ACCOUNT_IMAGES": "GET_ACCOUNT_IMAGES",
      "GET_ACCOUNT_IMAGES_INSTANCES": "GET_ACCOUNT_IMAGES_INSTANCES",
      "GET_ACCOUNT_INSTANCES": "GET_ACCOUNT_INSTANCES",
      "INVALID_ACCOUNT_WIZARD_STEP_ARN": "INVALID_ACCOUNT_WIZARD_STEP_ARN",
      "INVALID_ACCOUNT_WIZARD_STEP_POLICY": "INVALID_ACCOUNT_WIZARD_STEP_POLICY",
      "INVALID_ACCOUNT_WIZARD_STEP_ROLE": "INVALID_ACCOUNT_WIZARD_STEP_ROLE",
      "UPDATE_ACCOUNT": "UPDATE_ACCOUNT",
      "UPDATE_ACCOUNTS": "UPDATE_ACCOUNTS",
      "UPDATE_ACCOUNT_FIELD": "UPDATE_ACCOUNT_FIELD",
      "UPDATE_ACCOUNT_HIDE": "UPDATE_ACCOUNT_HIDE",
      "UPDATE_ACCOUNT_IMAGE": "UPDATE_ACCOUNT_IMAGE",
      "UPDATE_ACCOUNT_IMAGES_INSTANCES": "UPDATE_ACCOUNT_IMAGES_INSTANCES",
      "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL": "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL",
      "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP": "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP",
    },
    "applicationStatus": Object {
      "STATUS_5XX": "5XX",
      "default": "5XX",
    },
    "confirmationModal": Object {
      "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
      "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
    },
    "filter": Object {
      "GRAPH_OPENSHIFT_SET_FILTER_VALUE": "GRAPH_OPENSHIFT_SET_FILTER_VALUE",
      "GRAPH_RHEL_SET_FILTER_VALUE": "GRAPH_RHEL_SET_FILTER_VALUE",
      "TOOLBAR_ADD_FILTER": "TOOLBAR_ADD_FILTER",
      "TOOLBAR_CLEAR_DATE_TYPE": "TOOLBAR_CLEAR_DATE_TYPE",
      "TOOLBAR_CLEAR_FILTERS": "TOOLBAR_CLEAR_FILTERS",
      "TOOLBAR_REMOVE_FILTER": "TOOLBAR_REMOVE_FILTER",
      "TOOLBAR_SET_DATE_TYPE": "TOOLBAR_SET_DATE_TYPE",
      "TOOLBAR_SET_FILTER_TYPE": "TOOLBAR_SET_FILTER_TYPE",
      "TOOLBAR_SET_FILTER_VALUE": "TOOLBAR_SET_FILTER_VALUE",
      "TOOLBAR_SET_SORT_TYPE": "TOOLBAR_SET_SORT_TYPE",
      "TOOLBAR_TOGGLE_SORT_ASCENDING": "TOOLBAR_TOGGLE_SORT_ASCENDING",
    },
    "report": Object {
      "REPORT_EXPORT": "REPORT_EXPORT",
      "default": "REPORT_EXPORT",
    },
    "systemConfig": Object {
      "GET_SYSTEM_CONFIG": "GET_SYSTEM_CONFIG",
      "default": "GET_SYSTEM_CONFIG",
    },
    "toastNotifications": Object {
      "TOAST_ADD": "TOAST_ADD",
      "TOAST_PAUSE": "TOAST_PAUSE",
      "TOAST_REMOVE": "TOAST_REMOVE",
      "TOAST_RESUME": "TOAST_RESUME",
    },
    "user": Object {
      "USER_CREATE": "USER_CREATE",
      "USER_DELETE": "USER_DELETE",
      "USER_INFO": "USER_INFO",
      "USER_LOCALE": "USER_LOCALE",
      "USER_LOGIN": "USER_LOGIN",
      "USER_LOGOUT": "USER_LOGOUT",
      "USER_STORED_DATA": "USER_STORED_DATA",
      "USER_STORED_DATA_REMOVE": "USER_STORED_DATA_REMOVE",
    },
  },
  "reportTypes": Object {
    "REPORT_EXPORT": "REPORT_EXPORT",
    "default": "REPORT_EXPORT",
  },
  "systemConfigTypes": Object {
    "GET_SYSTEM_CONFIG": "GET_SYSTEM_CONFIG",
    "default": "GET_SYSTEM_CONFIG",
  },
  "toastNotificationTypes": Object {
    "TOAST_ADD": "TOAST_ADD",
    "TOAST_PAUSE": "TOAST_PAUSE",
    "TOAST_REMOVE": "TOAST_REMOVE",
    "TOAST_RESUME": "TOAST_RESUME",
  },
  "userTypes": Object {
    "USER_CREATE": "USER_CREATE",
    "USER_DELETE": "USER_DELETE",
    "USER_INFO": "USER_INFO",
    "USER_LOCALE": "USER_LOCALE",
    "USER_LOGIN": "USER_LOGIN",
    "USER_LOGOUT": "USER_LOGOUT",
    "USER_STORED_DATA": "USER_STORED_DATA",
    "USER_STORED_DATA_REMOVE": "USER_STORED_DATA_REMOVE",
  },
}
`;

exports[`ReduxTypes should have specific type properties: specific types 1`] = `
Object {
  "aboutModal": Object {
    "ABOUT_MODAL_HIDE": "ABOUT_MODAL_HIDE",
    "ABOUT_MODAL_SHOW": "ABOUT_MODAL_SHOW",
  },
  "account": Object {
    "ADD_ACCOUNT": "ADD_ACCOUNT",
    "ADD_ACCOUNT_SHOW": "ADD_ACCOUNT_SHOW",
    "ADD_ACCOUNT_WIZARD": "ADD_ACCOUNT_WIZARD",
    "ADD_ACCOUNT_WIZARD_STEP_ARN": "ADD_ACCOUNT_WIZARD_STEP_ARN",
    "ADD_ACCOUNT_WIZARD_STEP_POLICY": "ADD_ACCOUNT_WIZARD_STEP_POLICY",
    "ADD_ACCOUNT_WIZARD_STEP_ROLE": "ADD_ACCOUNT_WIZARD_STEP_ROLE",
    "DELETE_ACCOUNT": "DELETE_ACCOUNT",
    "DELETE_ACCOUNTS": "DELETES_ACCOUNTS",
    "DELETE_ACCOUNT_HIDE": "DELETE_ACCOUNT_HIDE",
    "DELETE_ACCOUNT_SHOW": "DELETE_ACCOUNT_SHOW",
    "EDIT_ACCOUNT_HIDE": "EDIT_ACCOUNT_HIDE",
    "EDIT_ACCOUNT_SHOW": "EDIT_ACCOUNT_SHOW",
    "GET_ACCOUNT": "GET_ACCOUNT",
    "GET_ACCOUNTS": "GET_ACCOUNTS",
    "GET_ACCOUNT_IMAGE": "GET_ACCOUNT_IMAGE",
    "GET_ACCOUNT_IMAGES": "GET_ACCOUNT_IMAGES",
    "GET_ACCOUNT_IMAGES_INSTANCES": "GET_ACCOUNT_IMAGES_INSTANCES",
    "GET_ACCOUNT_INSTANCES": "GET_ACCOUNT_INSTANCES",
    "INVALID_ACCOUNT_WIZARD_STEP_ARN": "INVALID_ACCOUNT_WIZARD_STEP_ARN",
    "INVALID_ACCOUNT_WIZARD_STEP_POLICY": "INVALID_ACCOUNT_WIZARD_STEP_POLICY",
    "INVALID_ACCOUNT_WIZARD_STEP_ROLE": "INVALID_ACCOUNT_WIZARD_STEP_ROLE",
    "UPDATE_ACCOUNT": "UPDATE_ACCOUNT",
    "UPDATE_ACCOUNTS": "UPDATE_ACCOUNTS",
    "UPDATE_ACCOUNT_FIELD": "UPDATE_ACCOUNT_FIELD",
    "UPDATE_ACCOUNT_HIDE": "UPDATE_ACCOUNT_HIDE",
    "UPDATE_ACCOUNT_IMAGE": "UPDATE_ACCOUNT_IMAGE",
    "UPDATE_ACCOUNT_IMAGES_INSTANCES": "UPDATE_ACCOUNT_IMAGES_INSTANCES",
    "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL": "UPDATE_ACCOUNT_IMAGE_FIELD_RHEL",
    "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP": "UPDATE_ACCOUNT_IMAGE_FIELD_RHOCP",
  },
  "applicationStatus": Object {
    "STATUS_5XX": "5XX",
    "default": "5XX",
  },
  "confirmationModal": Object {
    "CONFIRMATION_MODAL_HIDE": "CONFIRMATION_MODAL_HIDE",
    "CONFIRMATION_MODAL_SHOW": "CONFIRMATION_MODAL_SHOW",
  },
  "filter": Object {
    "GRAPH_OPENSHIFT_SET_FILTER_VALUE": "GRAPH_OPENSHIFT_SET_FILTER_VALUE",
    "GRAPH_RHEL_SET_FILTER_VALUE": "GRAPH_RHEL_SET_FILTER_VALUE",
    "TOOLBAR_ADD_FILTER": "TOOLBAR_ADD_FILTER",
    "TOOLBAR_CLEAR_DATE_TYPE": "TOOLBAR_CLEAR_DATE_TYPE",
    "TOOLBAR_CLEAR_FILTERS": "TOOLBAR_CLEAR_FILTERS",
    "TOOLBAR_REMOVE_FILTER": "TOOLBAR_REMOVE_FILTER",
    "TOOLBAR_SET_DATE_TYPE": "TOOLBAR_SET_DATE_TYPE",
    "TOOLBAR_SET_FILTER_TYPE": "TOOLBAR_SET_FILTER_TYPE",
    "TOOLBAR_SET_FILTER_VALUE": "TOOLBAR_SET_FILTER_VALUE",
    "TOOLBAR_SET_SORT_TYPE": "TOOLBAR_SET_SORT_TYPE",
    "TOOLBAR_TOGGLE_SORT_ASCENDING": "TOOLBAR_TOGGLE_SORT_ASCENDING",
  },
  "report": Object {
    "REPORT_EXPORT": "REPORT_EXPORT",
    "default": "REPORT_EXPORT",
  },
  "systemConfig": Object {
    "GET_SYSTEM_CONFIG": "GET_SYSTEM_CONFIG",
    "default": "GET_SYSTEM_CONFIG",
  },
  "toastNotifications": Object {
    "TOAST_ADD": "TOAST_ADD",
    "TOAST_PAUSE": "TOAST_PAUSE",
    "TOAST_REMOVE": "TOAST_REMOVE",
    "TOAST_RESUME": "TOAST_RESUME",
  },
  "user": Object {
    "USER_CREATE": "USER_CREATE",
    "USER_DELETE": "USER_DELETE",
    "USER_INFO": "USER_INFO",
    "USER_LOCALE": "USER_LOCALE",
    "USER_LOGIN": "USER_LOGIN",
    "USER_LOGOUT": "USER_LOGOUT",
    "USER_STORED_DATA": "USER_STORED_DATA",
    "USER_STORED_DATA_REMOVE": "USER_STORED_DATA_REMOVE",
  },
}
`;
